charts/openforms/values.yaml

```diff
@@ -443,16 +443,18 @@ worker:
   resources: {}
   # Defaults to 60s
   maxWorkerLivenessDelta: ""
-  livenessProbe:
+  livenessProbe: 
+    enabled: false
     exec:
       command:
-        - python
-        - /app/bin/check_celery_worker_liveness.py
+        - /bin/sh
+        - -c
+        - celery --workdir src --app openforms.celery inspect --destination celery@${HOSTNAME} active
```

Comment:
This is a bit to naive - the worker host name can be set via envvars and is especially useful when using different queues (short lived / prio / long-running tasks). The celery worker script in OF has:

```bash
QUEUE=${1:-${CELERY_WORKER_QUEUE:=celery}}
WORKER_NAME=${2:-${CELERY_WORKER_NAME:="${QUEUE}"@%n}}
```
which indeed defaults to `celery@$host`, but is not guaranteed to be that